soh/src/code/z_message_PAL.c

```diff
@@ -1662,6 +1662,21 @@ void Message_OpenText(GlobalContext* globalCtx, u16 textId) {
         } else if (gSaveContext.n64ddFlag && (textId == 0x7040 || textId == 0x7088)) {
             // rando hints at altar
             msgCtx->msgLength = font->msgLength = CopyAltarMessage(font->msgBuf, sizeof(font->msgBuf));
+        } else if (textId == 0x00b4 && CVar_GetS32("gInjectSkulltulaCount", 0) != 0 &&
```

Comment:
i think it might read cleaner to have the language check inside the first if statement, so
```
if (textId == 0x00b4 && CVar_GetS32("gInjectSkulltulaCount", 0) != 0) {
  if (gSaveContext.language == LANGUAGE_FRA) {
    strcpy
  }
}
```
or maybe use a switch/case instead